torch/jit/frontend.py

```diff
@@ -305,6 +305,32 @@ def get_default_args(fn):
     }
 
 
+def get_default_args_for_class(cls):
+    """
+    Get default arguments for all methods in a class (except for static methods).
+
+    Args:
+        cls: type - The class type to inspect for default arguments.
+    Returns:
+        A Dict[str, Dict[str, Any]] which maps each method name to a Dict[str, Any]
+        that maps each argument name to its default value.
+    """
+    # Get methods (except static methods because those are compiled separately as
```

Comment:
OOC, are default arg values for static methods already supported?

Reply:
Yeah, static methods of classes are actually compiled as if they were standalone methods (since there is no `self` argument or associated instance).